SWLOR.Game.Server/ChatCommand/SetScale.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using SWLOR.Game.Server.NWN;
+using SWLOR.Game.Server.ChatCommand.Contracts;
+using SWLOR.Game.Server.Enumeration;
+using SWLOR.Game.Server.GameObject;
+using SWLOR.Game.Server.NWN.Enum;
+
+namespace SWLOR.Game.Server.ChatCommand
+{
+    [CommandDetails("Sets Scale.", CommandPermissionType.DM | CommandPermissionType.Admin)]
+    public class SetScale : IChatCommand
+    {
+        public void DoAction(NWPlayer user, NWObject target, NWLocation targetLocation, params string[] args)
+        {
+
+            float value = float.Parse(args[0]);
+
+            _.SetObjectVisualTransform(target, ObjectVisualTransform.Scale, value);
```

Comment:
Also, this won't be persistent as far as I know. Don't know if that matters or not.

Reply:
Its not an issue atm, DM's should only be using this on NPC's until I can do much further in depth testing, but NPC use is fine.